charts/hono/example/add_example_data_device_registry.sh

```diff
@@ -11,7 +11,7 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #*******************************************************************************
-HTTP_BASE_URL="http://{{ .Release.Name }}-service-device-registry:8080/v1"
+HTTP_BASE_URL="http://{{ include "hono.fullname" . }}-service-device-registry:8080/v1"
```

Comment:
does this actually produce the desired outcome? Don't we need to use a `printf` statement here as well?